bundles/org.openhab.binding.pjlinkdevice/README.md

```diff
@@ -0,0 +1,75 @@
+# PJLink Binding
+
+This binding allows you to control devices with [PJLink](https://pjlink.jbmia.or.jp/english/) support. 
+PJLink is a protocol that was developed to standardize IP control of digital projectors, but can also be used by other device types.
+
+Aspects that can be controlled via PJLink are power on/off, input source selection and volume/mute setting.
+
+## Supported Things
+
+This binding supports devices which implement the PJLink protocol (Class 1). 
+
+Limitations at this point:
+
+- only IPv4 connections are supported
+- only PJLink Class 1 commands are supported. Class 2 devices should work fine nevertheless, it's just the Class 2 features that will not work.
```

Comment:
Please use a new line for each sentence

Reply:
I did so, but this triggers a code analysis warning:
```
[WARNING] .binding.pjlinkdevice/README.md:[15]
The line after a Markdown list must be empty.
```
I guess it's not possible to satisfy both constraints...

Reply:
Ok. Then better leave it like it is.